.github/workflows/ci.yaml

```diff
@@ -130,40 +130,13 @@ jobs:
         bgp-type: [native, frr]
         deployment: [manifests, helm]
     steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.9.1
-        with:
-          access_token: ${{ github.token }}
-
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
```

Comment:
isn't the checkout from the setup action enough?

Reply:
Nope, we need this otherwise we won't have the fresh setup action (and this PR will fail, and every time we change the setup action we won't have the fresh version)